frontend/src/components/common/DocumentUpload.vue

```diff
@@ -154,9 +142,16 @@ export default {
       this.setErrorAlert('Sorry, an unexpected error seems to have occurred. Try uploading your files later.');
     },
     async uploadFile(env) {
+      let fileExtensionValue;
+      if(this.smallFileExtension){
+        fileExtensionValue = getFileExtensionWithDot(this.file.name);
+      }else{
+        fileExtensionValue = this.file.type;
+      }
+
```

Comment:
There's no good place to put this. So I'm slotting it here. 

Not sure if it's out of scope for the ticket, but do we need to enforce fileRules ? i.e. <10mb and appropriate file types? 
https://github.com/bcgov/EDUC-STUDENT-PROFILE/blob/e34ffe3ef06ef160723692cef37bb710165f3b4b/frontend/src/components/DocumentUpload.vue#L215 <- can follow this pattern for the file rules. 

Also I think the ticket wants a hint for the file types that are accepted. Can leverage "hint" attribute in v-file-input for this feature. 
https://vuetifyjs.com/en/api/v-file-input/#props:~:text=undefined-,Hint%20text,-%23